MonoGame.Framework/Matrix.cs

```diff
@@ -715,7 +1008,16 @@ public static void CreatePerspectiveFieldOfView(float fieldOfView, float aspectR
 		    result.M43 = (nearPlaneDistance * farPlaneDistance) / (nearPlaneDistance - farPlaneDistance);
         }
 
-
+        /// <summary>
+        /// Creates a new projection <see cref="Matrix"/> for customized perspective view.
+        /// </summary>
+        /// <param name="left">Left coordinate of the viewing field.</param>
+        /// <param name="right">Right coordinate of the viewing field.</param>
+        /// <param name="bottom">Bottom coordinate of the viewing field.</param>
+        /// <param name="top">Top coordinate of the viewing field.</param>
+        /// <param name="nearPlaneDistance">Near plane of the viewing field.</param>
+        /// <param name="farPlaneDistance">Far plane of the viewing field.</param>
+        /// <returns>The new <see cref="Matrix"/> for customized perspective view.</returns>

```

Comment:
The repetition itself doesn't really bother me, but in this case it is not really the correct terminology.  `left`, `right`, `bottom` and `top` are the bounds of the x and y axes at the near plane.  `Viewing field` is not a term used in 3D rendering.  Without copying XNA docs exactly, something like the following would be better.

        /// <param name="left">Lower x-value at the near plane.</param>
        /// <param name="right">Upper x-value at the near plane.</param>

and

        /// <param name="nearPlaneDistance">Depth of the near plane.</param>